src/components/editor/Editor.tsx

```diff
@@ -1,35 +1,42 @@
 import React from "react";
-import AttributeEditor from "./attributeEditor/AttributeEditor";
-import ResourceProfileMapping from "./resourceProfileMapping/ResourceProfileMapping";
-import StructuredefSettings from "../structuredefSettings/StructuredefSettings";
+import AttributeEditor from "src/components/editor/attributeEditor/AttributeEditor";
+import ResourceProfileMapping from "src/components/editor/resourceProfileMapping/ResourceProfileMapping";
+import StructureDefSettings from "src/components/structureDefSettings/StructureDefSettings";
 import { useSelector } from "react-redux";
-import { RootState } from "../../state/store";
+import { RootState } from "src/state/store";
+import clsx from "clsx";
 import {
   IElementDefinition,
   IStructureDefinition
 } from "@ahryman40k/ts-fhir-types/lib/R4";
-import useStyles from "./style";
-import { Paper } from "@material-ui/core";
-import Navbar from "../navbar/Navbar";
+import useStyles from "src/components/editor/style";
+import {
+  Paper,
+  Container,
+  Typography,
+  Breadcrumbs,
+  Button
+} from "@material-ui/core";
+import Navbar from "src/components/navbar/Navbar";
 
 const Editor: React.FC<{}> = () => {
   const { loading, structureDefinition, selectedAttributeId } = useSelector(
     (state: RootState) => state.resource
   );
-  const structuredefSettings: string = "structuredefSettings";
   const classes = useStyles();
+  const splitedAttributeSelected = selectedAttributeId?.split(".");
   let attribute: IElementDefinition | undefined = undefined;
 
-  if (selectedAttributeId && selectedAttributeId !== structuredefSettings) {
+  if (selectedAttributeId && selectedAttributeId !== "structureDefSettings") {
```

Comment:
as discussed, re-organize the state so we can use a more structured way of handling "meta" updates and "element" updates